File: SoloLearn Codes/Argentina/Argentina.cs
using System;


namespace Sololearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int peso = Convert.ToInt32(Console.ReadLine());
            int dollar = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine(peso < dollar * 50 ? "Pesos" : "Dollars");
        }
    }
}

File: SoloLearn Codes/Argentina/Argentina.java
import java.util.Scanner;

public class Argentina
{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int peso = sc.nextInt();
        int dollar = sc.nextInt();
        System.out.println(peso < dollar * 50 ? "Pesos" : "Dollars");
    }
}